.github/workflows/wasm-sdk-documentation-check.yml

```
@@ -0,0 +1,152 @@
+          # Check if there are changes
+          if ! git diff --quiet docs.html docs_manifest.json AI_REFERENCE.md; then
+            git config --local user.email "github-actions[bot]@users.noreply.github.com"
+            git config --local user.name "github-actions[bot]"
+            git add docs.html docs_manifest.json AI_REFERENCE.md
+            git commit -m "chore: update WASM SDK documentation [skip ci]"
+            git push
+          fi
+          
+      - name: Create documentation update PR (protected branches)
+        if: github.event_name == 'push' && (contains(github.ref, 'master') || contains(github.ref, 'main')) && steps.doc-check.outputs.status == 'failure'
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)